app/views/reminder_mailings/show.html.slim

```diff
@@ -0,0 +1,54 @@
+= render 'section_navigation'
+
+h1 #{@reminder_mailing.t_enum(:kind)} Erinnerungs-Email
+
+h2 Versand Einzelheiten
+
+dl.dl-horizontal
+  dt Art
+  dd= @reminder_mailing.t_enum(:kind)
+  dt Erstellt am
+  dd= l(@reminder_mailing.created_at.to_date)
+  dt Status
+  - if @reminder_mailing.sending_triggered
+    dd Übermittelt
+  - else
+    dd Nicht versandt
+h3 Email Beispiel
+- example = @reminder_mailing.reminder_mailing_volunteers.sample.process_template
+h4 Betreff
+.mailing-body-preview= example[:subject]
+h4 Text
+.mailing-body-preview= markdown(example[:body])
+
+h3
+  - if @reminder_mailing.sending_triggered
+    'Versandt an
+  - else
+    'Wird versendet an
+
+table.table.table-condensed
+  thead
+    tr
+      th Freiwillige/r
+      th Einsatz
+  tbody
+    - @reminder_mailing.reminder_mailing_volunteers.each do |mailing_volunteer|
+      tr
+        td= link_to mailing_volunteer.volunteer.contact.full_name, mailing_volunteer.volunteer
+        td
+          - if mailing_volunteer.assignment?
+            = link_to mailing_volunteer.reminder_mailable.to_label, mailing_volunteer.reminder_mailable
+          - else
+            = link_to mailing_volunteer.reminder_mailable.group_offer.to_label, mailing_volunteer.reminder_mailable.group_offer
+
+
+nav.navbar.hidden-print
+  ul.list-inline
+    - if !@reminder_mailing.sending_triggered
+      li= button_link 'Emails versenden', initiate_mailing_reminder_mailing_path(@reminder_mailing)
+      li= button_link 'Ändern', edit_reminder_mailing_path(@reminder_mailing)
+      li= link_to 'Löschen', reminder_mailing_path(@reminder_mailing), confirm_deleting(@reminder_mailing, 'btn btn-default')
+      li= button_link 'Zurück', edit_reminder_mailing_path(@reminder_mailing)
```

Comment:
Do we need both the "Ändern" and "Zurück" buttons? They both point to the same page.

Reply:
Zurück will be dropped.